services/authenticator.js

```diff
@@ -140,6 +147,32 @@ function Authenticator() {
       });
   };
 
+  this.logout = async (opts) => {
+    const path = `${os.homedir()}/.lumberrc`;
+
+    return new P((resolve, reject) => {
+      fs.stat(path, (err) => {
+        if (err === null) {
+          fs.unlinkSync(path);
+
+          if (opts.log) {
+            console.log(chalk.green('👍  You\'re now unlogged 👍 '));
+          }
+
+          resolve();
+        } else if (err.code === 'ENOENT') {
+          if (opts.log) {
+            logger.error('🔥  You\'re not logged 🔥');
+          }
```

Comment:
Why do we have sometimes logger and sometimes console, it's confusing ?

Reply:
`console.error is used to print the full stack trace`

Reply:
Yeah I know but what I mean is why don't we migrate it to logger ? And why don't we use `logger.success` for example to print a success message instead of using `console.log` with `charn`